test/test-apps3buckets-delete.js

```diff
@@ -0,0 +1,40 @@
+const { assert } = require('chai');
+const nock = require('nock');
+
+const config = require('../app/config');
+const handlers = require('../app/apps3buckets/handlers');
+
+
+describe('Edit bucket form', () => {
+  describe('when revoking access to user', () => {
+    it('make request to API', () => {
+      const apps3bucket_id = 42;
+      const redirect_to = 'apps/123';
+
+      // Mock `DELETE /apps3buckets/:id` request
```

Comment:
This comment doesn't add any information. Comments are to explain _why_ the code is doing something, not _what_ it is doing.